# Furnace energy use
As it's complicated to find exact data for plastic use, I am going to use as a concept how much plastic a human uses if his water intake was all from plastic or aluminium per day. 
An important finding is that due to Stefan Boltzmann law the size of the chamber needs to be minimal around 1 liter without some special coating so it needs compression

In [34]:
can_weight_gr = 14 # 330 ml aluminium can
al_can_vol = 330
plastic_bottle_gr = 11 # 500 ml plastic bottle
plastic_bottle_vol = 500
water_human_gr = 3700 # water use per day ml male
plastic_per_day = water_human_gr * plastic_bottle_gr / plastic_bottle_vol
aluminium_per_day = water_human_gr * can_weight_gr / al_can_vol
aluminium_per_week = aluminium_per_day * 7
alum_increase = aluminium_per_day - plastic_per_day
percent_increase = alum_increase * 100 / plastic_per_day

print("Imaginary daily plastic use (gr):{} Versus aluminum use (gr) :{:.1f} A mass increase of (%):{:.1f}".format(plastic_per_day,aluminium_per_day,percent_increase))
print("Imaginary weekly aluminium use (gr):{:.1f}".format(aluminium_per_week))

Imaginary daily plastic use (gr):81.4 Versus aluminum use (gr) :157.0 A mass increase of (%):92.8
Imaginary weekly aluminium use (gr):1098.8


In [70]:
# Energy required to melt aluminium in ideal conditions

ar_alum = 27 # Aluminium Ar
moles = aluminium_per_week / ar_alum # Moles of aluminium per week
molar_heat_j = 24.20 # Molar Heat capacity joule / K * mole
molar_fusion = 10710 # Fusion energy joule /  mole
temp_celsius = 19 # Mean home temperature 19 C
temp_mean = temp_celsius + 273.15
temp_melting_point_celsius = 660.32 # Temperature of aluminium melting point
temp_melting_point_kelvin = temp_melting_point_celsius + 273.15
temp_delta = temp_melting_point_kelvin - temp_mean # Temperature delta between home and melting point  
energy_to_temp = temp_delta * molar_heat_j * moles# Energy to reach that temperature joules
energy_to_melt = molar_fusion * moles # Energy to melt
energy_without_loses = energy_to_temp + energy_to_melt # Total energy without loses
energy_without_loses_kwh = energy_without_loses / (60*60*1000)# Joules to KWh

#Container
# Heating container made of 1 mm steel 
# Definately needs compression before heating as radiative heating creates an upper limit
area_of_steel = 10 * 10 * 6 / (100*100) # 6 sides
# volume of container = 1 litre 
weight_1mm = 7850*2 # 2m^2 steel weight 1mm thickness
weight_of_steel = area_of_steel * weight_1mm 
ar_steel = 56# Iron Ar
molar_heat_steel_j = 25.10 # Molar Heat capacity joule / K * mole
moles_steel = weight_of_steel / ar_steel # Moles of iron
energy_to_temp_steel = temp_delta * molar_heat_steel_j * moles_steel # Energy to heat iron
energy_container_kwh = energy_to_temp_steel / (60*60*1000) # Energy in kwh container


#Radiative loses
boltz = 5.670373 * 10**(-8) #Boltzmann constant Watts/(Meter*Kelvin^4)
e = 1 #Emissivity 1
rad_power = e * area_of_steel * boltz * ((temp_melting_point_kelvin**4)-(temp_mean**4)) #Power loss
print("Peak power loss :{:.0f}".format(rad_power),)
#Energy to melt aluminium
print("Energy to melt aluminium (per week)(without loses): {:.0f} Joules or {:.3f} KWh".format(energy_without_loses,energy_without_loses_kwh))
#Energy to heat container
print("Energy to heat steel (per week)(without loses): {:.0f} or {:.3f}KWh".format(energy_to_temp_steel,energy_container_kwh))
print("Black body radiation Watts at peak temperature")#Radiative loses


watts = 5000 # Power input Watts
joule_sec = watts * 1 # joules per second
time_run = 1000 # seconds max run
joules_per_kelvin = molar_heat_steel_j * moles_steel + molar_heat_j * moles # up to melting temp 660 

temp = temp_mean
for i in range(time_run):
    rad_power_2 = e * area_of_steel * boltz * ((temp**4)-(temp_mean**4))
    
    if temp < temp_melting_point_kelvin:
        joule_sec_2 = joule_sec - rad_power_2
        kelvin_increase = joule_sec_2 / joules_per_kelvin
        temp = temp + kelvin_increase
    else:
        time = i
        break
time2 = energy_to_melt / joule_sec_2 
total = time + time2
print("Time to heat",time,"Time at Temperature {:.0f}".format(time2),"Total time {:.0f}".format(total))
energy_usage = (total * watts)/(60*60*1000)
print("Total energy KWh {:.3f}".format(energy_usage))


Peak power loss :2558
Energy to melt aluminium (per week)(without loses): 1067450 Joules or 0.297 KWh
Energy to heat steel (per week)(without loses): 270777 or 0.075KWh
Black body radiation Watts at peak temperature
Time to heat 219 Time at Temperature 178 Total time 397
Total energy KWh 0.552


This furnace would have two limits: One would be its surface area as due to boltzmann law it has to be minimized so its volume would have to be around a litre (1L) unless, special silver polished coatings would be used to decrease the emissivity. The second would be the need to have high peek power around a 5 KW to reach temperature fast and melt the aluminium. At less than 2.5 kw without coating it would be impossible to work.
It's total energy would be then around 0.55 KWh which is around half a shower use, which would make it viable. 
Still the need of a aluminium press would complicate its construction, and make it more expensive.